.gitignore

```diff
@@ -1,6 +1,8 @@
 _bin/
 _output/
 .python_virtual_env
+/yarn.lock
```

Comment:
I suggest to remove this line, to version control the `yarn.lock` file;
please see https://github.com/yarnpkg/yarn/issues/1583

Reply:
When we are building images or running deck locally, we use yarn to generate the node_modules and yarn.lock and run `hack/ts-rollup` to create the final js files that go in the container. When the build fails, we get a dirty git tree but these folders should never be committed.

Reply:
I am not objecting to, actually am agreeing to, add 'node_modules' directory to gitignore.
What I am suggesting is that we SHOULD version control the 'yarn.lock' file, and it's just a single file, not a directory.